tendrl/commons/objects/definition/master.yaml

```diff
@@ -102,6 +104,16 @@ namespace.tendrl:
       list: /clusters
       value: clusters/$TendrlContext.integration_id
       atoms:
+        ConfigureMonitoring:
+          enabled: true
+          inputs:
+            mandatory:
+              - "Node[]"
```

Comment:
I dont think any input is required for this atom, you can simply use node_context and tendrl_context and NS.config